src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int index = 0;
+        try {
+            int[] sumArray = readSumFromFile(fromFileName);
+            writeInFile(sumArray[index], sumArray[index + 1], toFileName);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
```
divide your logic into 3 methods